doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -17,7 +17,13 @@ ifdef::PAM[]
 +
 [IMPORTANT]
 ====
-In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality deployment using the operator is for Technology Preview only. For more information about Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]. For a fully supported High Availability deployment, use the high availability authoring template on {OPENSHIFT} version 3.11. For instructions about deploying this template, see
```

Comment:
Is the capitalized "High Availability deployment" intentionally proper compared to the other lowercased instances here? Just curious if the HA deployment is a proper name or something.